_zip/missingapi.yml

```diff
@@ -19,7 +19,7 @@ references:
   name.vb: ToString
   fullName: XDocument.ToString
   fullName.vb: XDocument.ToString
-- uid: System​.Runtime​.CompilerServices.IndexerNameAttribute
+- uid: System.Runtime.CompilerServices.IndexerNameAttribute
```

Comment:
This isn't a missing API: https://docs.microsoft.com/en-us/dotnet/api/system.runtime.compilerservices.indexernameattribute

Reply:
#10829
Advanced (Windows Presentation Foundation)
2 minutes to read • Edit Online
In ThisSection
This section describes some of theadvanced areas in WPF.
WPF Architecture
XAML in WPF
BaseElement Classes
ElementTreeand Serialization
WPF Property System
Events in WPF
Input
Drag and Drop
Resources
Documents
Globalization and Localization
Layout
Migration and Interoperability
Performance
Threading Model
Unmanaged WPF API Reference